lib/ethereum.dart

```diff
@@ -57,15 +57,45 @@ class Ethereum {
 class RequestParams {
   external String get method;
 
-  external List<dynamic> get params;
+  external dynamic get params;
 
   // Must have an unnamed factory constructor with named arguments.
-  external factory RequestParams({String? method, List<dynamic>? params});
```

Comment:
Isn't this almost always a List? What was your use case to change it?

Reply:
I've tried it with list and it raise an error on js side. Not sure Im doing it right or not but in EIP it use the object as the param not List of object.